src/makeCalculator.js

```diff
@@ -4,7 +4,27 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+  calculator.add = (number) => (calculator.result += number);
+  calculator.subtract = (number) => (calculator.result -= number);
+  calculator.multiply = (number) => (calculator.result *= number);
+  calculator.divide = (number) => (calculator.result /= number);
```

Comment:
There is a potential issue with the `divide` method. If `number` is zero, this will result in a division by zero, which could cause unexpected behavior or errors. Consider adding a check to handle this case appropriately.